include/lsst/afw/image/Exposure.h

```diff
@@ -150,9 +150,9 @@ class Exposure : public lsst::daf::base::Persistable,
 
     // Get Members
     /// Return the MaskedImage
-    MaskedImageT getMaskedImage() { return _maskedImage; }
+    MaskedImageT & getMaskedImage() { return _maskedImage; }
     /// Return the MaskedImage
-    MaskedImageT getMaskedImage() const { return _maskedImage; }
+    MaskedImageT const & getMaskedImage() const { return _maskedImage; }
```

Comment:
Three shared pointers isn't all that small IMO, at least for an attribute we get all the time.  I believe we may it a by-value return originally to avoid dangling-pointer problems with Swig, so it seemed like the time to revert that.